src/main/cpp/RobotContainer.cpp

```diff
@@ -1,24 +1,29 @@
-/*----------------------------------------------------------------------------*/
```

Comment:
Keep copyright headers in your code - Needs to be in each file per Apache licensing rules